func1.js

```diff
@@ -0,0 +1,15 @@
+const runningAverage = () => {
+    let counter = 0;
+    let number = 0;
```

Comment:
The name of variables a little bit puzzled me on this line: `return counter/number;` (line 8)
Because `counter` is more suitable for counting of the function call and `number` better suites to the current sum which we have. 
So I would propose to swap these variables :)